scripts/test_unit.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 # test_unit.sh
+
+source cmd/proxy/.env
```

Comment:
if you include changes requested up there (.env) please make sure that `.env` is sourced before buffalo download starts. ideally at the beginning of `set-dev-env`
also there is a PR in focusing on this so you might get rid of it entirely as well #449

Reply:
I think it's an unintentional result of a rebase, not a direct change from this PR